manifests/base/training_job_editor_role.yaml

```diff
@@ -2,12 +2,15 @@
 apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Are we going to use these additional roles other than base [ClusterRole](https://github.com/kubeflow/tf-operator/blob/e358768661156c3ff1b8b22126090f6dae2374b8/manifests/base/cluster-role.yaml) ?
Currently, we use only base ClusterRole in the [Kustomize manifests](https://github.com/kubeflow/tf-operator/blob/e358768661156c3ff1b8b22126090f6dae2374b8/manifests/base/kustomization.yaml#L10).

Reply:
Agree, initial assumption was that we will use the base ClusterRole only hence I did not include others. why do we need this?

Reply:
Agree. Let me clean it up